Gemfile.lock

```diff
@@ -0,0 +1,251 @@
+PATH
```

Comment:
AFAIK this shouldn't really be in here for libraries as they should install the latest thing that's permitted by the `Gemfile` (as that's what'll happen in apps who use it). It's apps that should have the lockfile.